metpy/io/tests/test_metar.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) 2008,2015,2016,2017,2018,2019 MetPy Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+"""Test various metars."""
+from datetime import datetime
+
+import numpy as np
+from numpy.testing import assert_almost_equal, assert_equal
+
+from metpy.cbook import get_test_data
+from metpy.io import metar, surface_station_data
+
+
+def test_station_id_not_in_dictionary():
+    """Test for when the METAR does not correspond to a station in the dictionary."""
+    df = metar.parse_metar_to_pandas('METAR KLBG 261155Z AUTO 00000KT 10SM CLR 05/00 A3001 \
+    RMK AO2=')
+    assert df.station_id.values == 'KLBG'
+    assert_almost_equal(df.latitude.values, np.nan)
+    assert_almost_equal(df.longitude.values, np.nan)
+    assert_almost_equal(df.elevation.values, np.nan)
+    assert_almost_equal(df.wind_direction.values, 0)
+    assert_almost_equal(df.wind_speed.values, 0)
+
+
+def test_broken_clouds():
+    """Test for skycover when there are broken clouds."""
+    df = metar.parse_metar_to_pandas('METAR KLOT 261155Z AUTO 00000KT 10SM BKN100 05/00 A3001 \
+    RMK AO2=')
+    assert df.skyc1.values == 'BKN'
+    assert df.cloudcover.values == 6
+
+
+def test_few_clouds_():
+    """Test for skycover when there are few clouds."""
+    df = metar.parse_metar_to_pandas('METAR KMKE 261155Z AUTO /////KT 10SM FEW100 05/00 A3001 \
+    RMK AO2=')
+    assert df.skyc1.values == 'FEW'
+    assert df.cloudcover.values == 2
+    assert_almost_equal(df.wind_direction.values, np.nan)
+    assert_almost_equal(df.wind_speed.values, np.nan)
+
+
+def test_all_weather_given():
+    """Test when all possible weather slots are given."""
+    df = metar.parse_metar_to_pandas('METAR RJOI 261155Z 00000KT 4000 -SHRA BR VCSH BKN009 \
+    BKN015 OVC030 OVC040 22/21 A2987 RMK SHRAB35E44 SLP114 VCSH S-NW P0000 60021 70021 \
+    T02220206 10256 20211 55000=')
+    assert df.station_id.values == 'RJOI'
+    assert_almost_equal(df.latitude.values, 34.14, decimal=2)
+    assert_almost_equal(df.longitude.values, 132.24, decimal=2)
+    assert df.current_wx1.values == '-SHRA'
+    assert df.current_wx2.values == 'BR'
+    assert df.current_wx3.values == 'VCSH'
+    assert df.skyc1.values == 'BKN'
+    assert df.skylev1.values == 900
+    assert df.skyc3.values == 'OVC'
+    assert df.skylev3.values == 3000
+
+
+def test_missing_temp_dewp():
+    """Test when missing both temperature and dewpoint."""
+    df = metar.parse_metar_to_pandas('KIOW 011152Z AUTO A3006 RMK AO2 SLPNO 70020 51013 \
```

Comment:
Don't wrap the strings with `\`, but instead just put a closing quote on this line and open a new one on the next line. Since they're within the `()` for the function call, Python will automatically concatenate them. Also, would appreciate lining things up when you wrap to a new line. So like this:
```python
def = metar.parse_metar_to_pandas('KIOW 011152Z AUTO A3006 RMK AO2 SLPNO 70020 51013 '
                                                              'PWINO=')
```

It looks like you have several tests that do this. Fixing it up will probably fix your windows builds.